.github/workflows/_swift.yml

```
@@ -41,7 +41,7 @@ jobs:
       - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
       - run: sudo xcode-select --switch "$(find /Applications -maxdepth 1 -name "Xcode*${XCODE_VERSION}*.app" | sort -V | tail -n 1)"
       - name: Check Swift formatting
-        run: make format-check
+        run: find . -name "*.swift" -not -path "./FirezoneNetworkExtension/Connlib/Generated/*" -not -path "./FirezoneKit/.build/*" | xargs swift format lint --parallel --strict
```

Might be a little less brittle?

```suggestion
        run: git ls-files "*.swift" | xargs swift format lint --parallel --strict
```

---

changed and moved into a file based mise task (that can be called directly from this location)